lib/const.rb

```diff
@@ -89,6 +89,8 @@
 
 # Downloader maximum retry count
 CREW_DOWNLOADER_RETRY = ( ENV['CREW_DOWNLOADER_RETRY'].to_s.empty? ) ? 3 : ENV['CREW_DOWNLOADER_RETRY'].to_i
+# Downloader show progress bar or not
+CREW_DOWNLOADER_SHOW_PROGBAR = ENV.fetch('CREW_DOWNLOADER_SHOW_PROGBAR', '1').eql?('1')
```

Comment:
Shouldn't this default to 1?  The initial value will be nil (ie. same as zero or false) if the environment variable is not set.  Maybe you can change to `CREW_HIDE_PROGBAR` instead?